Specs/Core/joinUrlsSpec.js

```diff
@@ -108,7 +108,7 @@ defineSuite([
 
     it('appends qualfied url with path correctly to qualified url with path', function() {
         var result = joinUrls(qualifiedUrlWithPath, qualifiedUrlWithPath);
-        expect(result).toEqual(qualifiedUrlWithPath);
+        expect(result).toEqual(qualifiedUrlWithPath+absolutePath);
```

Comment:
This is really nit picky, but can you please add spaces around the `+`?  Thanks!